src/sasjs-help/index.js

```diff
@@ -101,18 +101,38 @@ export async function printHelpText() {
       'context <command>'
     )} - performs operations on contexts.
         * ${chalk.cyanBright('create')} - creates new context.
-          1) Provide target name.
-          2) Provide path to context config file (eg. ./contextConfig.json).
+          command example: sasjs create -source ../contextConfig.json -target targetName
+          NOTE: Providing target name (-target targetName) is optional. Default target name will be used if target name was omitted.
+
+          context config example:
+            {
+              "name": "New context",
+              "launchName": "SAS Job Execution launcher context",
+              "sharedAccountId": "cas",
+              "autoExecLines": ["%put hello;"],
+              "authorizedUsers": []
+            }
+          NOTE: Keys "name", "launchName", "sharedAccountId", "autoExecLines", "authorizedUsers" are required.
```

Comment:
are we sure that autoExecLines is required?

Reply:
according addapter's function, yes
![image](https://user-images.githubusercontent.com/25773492/93459500-76f49e80-f8ea-11ea-90ba-102cd2778552.png)